codegen/module_system.go

```diff
@@ -690,6 +691,16 @@ func (g *EndpointGenerator) generateEndpointFile(
 			strings.Title(method.Name) + "Endpoint"
 	}
 
+	// TODO: http client needs to support multiple thrift services
+	var clientMethodName string
+	if method.DownstreamMethod != nil {
```

Comment:
The usage of `clientMethodName` is guarded by a downstream existence [check](https://github.com/uber/zanzibar/pull/103/files/b135fde02cc527dfafa160c6810557a624660d88#diff-2426caa827592efd9d55743dd4943e3cR118), it's only used when downstream exists.